pkg/controlplane/apiserver/options/options.go

```diff
@@ -157,7 +157,8 @@ func (s *Options) AddFlags(fss *cliflag.NamedFlagSets) {
 
 	fs.BoolVar(&s.EnableLogsHandler, "enable-logs-handler", s.EnableLogsHandler,
 		"If true, install a /logs handler for the apiserver logs.")
-	fs.MarkDeprecated("enable-logs-handler", "This flag will be removed in v1.33") //nolint:errcheck
+	fs.MarkDeprecated("enable-logs-handler", "Log handler functionality is deprecated") //nolint:errcheck
```

Comment:
?
The log handler functionality is going away?
or ... this particular flag is going away?

Reply:
neither is planned to be removed at this point, but is disabled by default and marked deprecated to discourage new use